core/src/main/java/nl/nn/adapterframework/core/PipeLine.java

```diff
@@ -396,10 +396,7 @@ public void configure(IPipe pipe) throws ConfigurationException {
 			pipeStatistics.put(pipe.getName(), new StatisticsKeeper(pipe.getName()));
 			//congestionSensors.addSensor(pipe);
 		} catch (Throwable t) {
-			if (t instanceof ConfigurationException) {
-				throw (ConfigurationException)t;
-			}
-			throw new ConfigurationException("Exception configuring Pipe ["+pipe.getName()+"]",t);
+			throw new ConfigurationException("Exception configuring "+ ClassUtils.nameOf(pipe) +" ["+pipe.getName()+"]",t);
```

Comment:
Could you check other locations where pipes are configured, like for validators and wrappers, to see the same extension to the exception message is made?

Reply:
You just showed me it is already done!